site/tl.njk

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+meta_title: Kilusan para mapatigil ang Pandemyang COVID-19
+meta_description: Hindi kaya ng mga pamahalaan natin na gawin itong mag-isa, kailangan nating kumilos ngayon! Gawin natin ang ating bahagi at manatili sa mga bahay natin, gago!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Kilusan para mapatigil ang Pandemyang COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Ang mga pamahalaan natin ay mahinay na nagpapatupad ng mga hakbang para kontrolin ang 
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                at pigilan ang <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">Pandemyang COVID-19</a>.
+                Mabagal ng reaksyon, mga patakaran ng pagpayapang pampubliko, at ang kanilang pag-pilit na patatagin ang ekonomiya ay nagpapatigil sa kanila ng 
+                pagbibigay ng mga kailangang hakbang para maprotektahan ang mga milyon-milyong mula sa sakit na ito. Gayunpaman, hindi lang ito ang pasan ng gobyerno. 
+                Oras na natin, bilang mamamayan ng daigdig, para kumilos ngayon at gawin ang ating bahagi para labanan ang COVID-19
+                <br /><br />
+                Ilagay natin ito sa matuwid: <strong>Manatili sa mga bahay natin, Gago!</strong>
+            </p>
+
+            <h2 class="text-3xl">Ang Manipestong Pag-Quarantine sa Sarili </h2>
+
+            <p>
+                Nang walang pinagaralan ng paggamot at walang makakakuha ng gumaganang bakuna para sa kahit isang taon pa, ang mabisa na paraan para
+                kontrolin ang pandemyang coronavirus ay bigyan ang virus ng mas kaunting pagkakataon na kumalat. Ito ay mga listahan ng mga aksyon, sa uri ng
+                madaling ipatupad sa mas mabisa sa paglaban ng pandemya, ay dapat na sapat bilang isang nakatakdang ng maluwag na patnubay para sa mga tao na gustong sumali
+                sa kilusan at gumawa ng aksyon na maaring talagang gumawa ng pagkakaiba.
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Huwag matakot, pero maging alerto.</li>
+                <li class="pb-2">Maghugas ng kamay palagi at magsanay ng magandang kaugalian sa pag-ubo at pagbahin.</li>
+                <li class="pb-2">Subukang hawakan ang iyong mukha nang kaunti hangga't maaari, kasali ang iyong bunganga, ilong, at mata.</li>
```

Comment:
Mas ok siguro kung **Subukang limitahan and paghawak sa iyong mukha...** ung gagamiting statement. Nung unang basa ko kasi akala ko kailangang hawakan ko kahit paminsan minsan ung muka ko.